bundles/org.openhab.binding.groupepsa/src/main/resources/OH-INF/i18n/groupepsa_fr.properties

```diff
@@ -0,0 +1,21 @@
+# binding
```

Comment:
Please note that we have a problem with this binding in Crowdin.
Lot of entries (channels) are missing (not proposed for translation).

Reply:
I have uploaded manually the English string in Crowdin.